### DISCUSSION TITLE
「开源之书·共读」#40：《维基经济学：大规模协作如何改变一切》 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#51] [Announcements]
# 「开源之书·共读」#40：《维基经济学：大规模协作如何改变一切》 - 「开源之道」

https://opensourceway.community/posts/book-of-open-source/40th-wikinomics/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实吧, wikipedia 到现在, 主要使命就是活着了...

要说, 大规模协作, 中国自古就有卓绝世界的经验, 
早在3千年前, 就已经能熟练的组织几十万人的军事/建造工程了...

到解放后, 更加炉火纯青, 动不动上百万人的大会战,战天斗地, 无所不能...

所以, 我们对大规模协作, 并没什么崇敬之情, 
因为, 在我们的经验中, 协作规模越大, 每个人越不自由, 和最终成果越无关联...

观察生存10年以上的成功 FLOSS/自由,开源项目, 应该可以感知到,
能生存下来的都是有 `仁慈的暴君`BDFL/Benevolent Dictator for Life 长期把持的,
绝对民主的, 很快, 每个人都感觉自己没有获得充分尊重,而只能离开...

所以, 西方现代的大规模协作, 一直是松散的,
Wikipedia 才是首次在线大规模密集/实时协作...

改变的可能不多:

- 证明我们可以
- 感觉, 我们不愿意
- 以后, 还是怎么舒服怎么来
